packages/synapse-react-client/src/components/EntityHeaderTable/EntityHeaderTable.tsx

```diff
@@ -357,58 +397,42 @@ export const EntityHeaderTable = (props: EntityHeaderTableProps) => {
-                                      backgroundColor: 'none',
-                                      float: 'right',
-                                      marginRight: '5px',
-                                    }}
-                                  />
-                                ),
-                                desc: (
-                                  <IconSvg
-                                    icon={'sortDown'}
-                                    wrap={false}
-                                    sx={{
-                                      color: 'primary.main',
-                                      backgroundColor: 'none',
-                                      float: 'right',
-                                      marginRight: '5px',
-                                    }}
-                                  />
-                                ),
-                              }[header.column.getIsSorted() as string] ?? null}
-                            </div>
-                            {header.column.getCanFilter() &&
-                            totalRowCount > UNMANAGEABLE_SUBJECT_COUNT ? (
-                              <div>
-                                <Filter column={header.column} table={table} />
-                              </div>
-                            ) : null}
-                          </>
+                          <Box display={'flex'} alignItems={'center'}>
+                            {flexRender(
+                              header.column.columnDef.header,
+                              header.getContext(),
+                            )}
+                            <Box mx={'auto'} />
+                            {header.column.getCanSort() && (
+                              <IconButton
+                                onClick={header.column.getToggleSortingHandler()}
+                                size={'small'}
+                                sx={{
+                                  marginLeft: 'auto',
+                                  marginRight: '16px',
+                                }}
+                              >
+                                <IconSvg
+                                  icon={
+                                    header.column.getIsSorted() === 'asc'
+                                      ? 'sortUp'
+                                      : 'sortDown'
+                                  }
+                                  wrap={false}
+                                  fontSize={'inherit'}
+                                  sx={{
+                                    color: header.column.getIsSorted()
+                                      ? 'primary.main'
+                                      : 'grey.700',
+                                    backgroundColor: 'none',
+                                  }}
+                                />
+                              </IconButton>
+                            )}
```

Comment:
If `getCanSort()` is true, always show the sort icon, changing the color if `getIsSorted` is true.